tutorials/source-modeling/plot_background_freesurfer_mne.py

```diff
@@ -0,0 +1,425 @@
+# transformation sets the center of its coordinate frame in the middle of the
+# conformed volume dimensions (``N / 2.``) with the axes oriented along the
+# axes of the volume itself. For more information, see
+# :ref:`coordinate_systems`.
+#
+# Since MNE-Python uses FreeSurfer extensively for surface computations (e.g.,
+# white matter, inner/outer skull meshes), internally MNE-Python uses the
+# Freeurfer surface RAS coordinate system (not the :mod:`nibabel` scanner RAS
+# system) for as many computations as possible, such as all source space
+# and BEM mesh vertex definitions.
+#
+# .. note::
+#       Whenever you see "MRI coordinates" or "MRI coords" in MNE-Python's
+#       documentation, you should assume that we are talking about the
+#       "FreeSurfer MRI surface RAS" coordinate frame!
+#
+# We can do similar computations as before to figure out where the given voxel
+# and RAS point is in FreeSurfer MRI coordinates (i.e., what we call just
+# "MRI coordinates" everywhere else in MNE):
+
+Torig = t1.header.get_vox2ras_tkr()
+print(t1.affine)
+print(Torig)
+xyz_mri = apply_trans(Torig, vox)
+imshow_mri(data, t1, vox, dict(MRI=xyz_mri), 'MRI slice')
+
+###############################################################################
+# Knowing these relationships and being mindful about transformations, we
+# can get from a point in any given space to any other space. Let's start out
+# by plotting the Nasion on a saggital MRI slice:
+
+fiducials = mne.coreg.get_mni_fiducials(subject, subjects_dir=subjects_dir)
+nasion_mri = [d for d in fiducials if d['ident'] == FIFF.FIFFV_POINT_NASION][0]
+print(nasion_mri)  # note it's in MRI coords
+
+###############################################################################
+# When we print the nasion, it displays as a ``DigPoint`` and shows its
+# coordinates in millimeters, but beware that the underlying data is
+# :ref:`actually stored in meters <units>`,
+# so before transforming and plotting we'll convert to millimeters:
+
+nasion_mri = nasion_mri['r'] * 1000  # meters → millimeters
+nasion_vox = np.round(
+    apply_trans(np.linalg.inv(Torig), nasion_mri)).astype(int)
+imshow_mri(data, t1, nasion_vox, dict(MRI=nasion_mri),
+           'Nasion estimated from MNI transform')
+
+###############################################################################
+# We can also take the digitization point from the MEG data, which is in the
+# "head" coordinate frame.
+#
+# .. note::
+#      The head coordinate frame in MNE is the "Neuromag" head coordinate
+#      frame. The origin is given by the intersection between a line connecting
+#      the LPA and RPA and the line orthogonal to it that runs through the
+#      nasion. It is also in RAS orientation, meaning that +X runs through
+#      the RPA, +Y goes through the nasion, and +Z is orthogonal to these
+#      pointing upward. See :ref:`coordinate_systems` for more information.
```

Comment:
This info is important enough that I don't see why it's called out as a note.  I'd integrate it with the sentence that precedes it.

Reply:
I recently learned that `.. note` is supposed to be for calling out important information

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes

I guess we could / should adopt this practice, and use `.. sidebar` for things that are asides / less important

Reply:
hmm.  The [official rST docs](https://docutils.sourceforge.io/docs/ref/rst/directives.html#note) don't actually give guidance on the semantics of the different admonition types, but it looks like [the sphinx docs do](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-note). TIL.  I guess we could argue about how applicable that guidance is for *tutorials* as opposed to *APIs*, but it doesn't seem important enough to bother; it's probably better to decide on guidelines that make sense for our project, document them, and stick to them.

Reply:
I would vote for `.. note` as being for important things, `.. sidebar` for things that are less important / asides, and `.. warning` for things that will likely trip up users. I'm not sure the extent to which we have used `.. note` this way previously or not

Reply:
Some observations:

- IMO the main text of a docstring or tutorial ought to contain important information, so saying that `.. note::` is for "important things" doesn't actually clarify what should go in a note and what should be part of the main text.
- we've had complaints (I think from @jasmainak) about too many notes breaking up the flow of the page and making it harder to read.  I am generally sympathetic with that view.

WDYT about demote notes to be semantically like sidebars?  Maybe we could get by with only one of them, or maybe we would want to only use `.. note::` in docstrings (so they render inline), and only using `.. sidebar::` in examples/tutorials (where the generally longer prose passages makes sidebars look decent)?  We could define both of them as "an aside, containing useful information that is not strictly needed to use the function / follow the tutorial".

Reply:
This one can be a sidebar because the focus is not really on the head coord frame. But the one about "MRI coordinates" in MNE really is important. I'll just make that one bold for now.